polkadex-primitives/src/lib.rs

```diff
@@ -121,4 +122,30 @@ impl OpenFinexUri {
     }
 }
 
+#[derive(Debug, Encode, Decode, PartialEq, PartialOrd, Ord, Eq, Clone)]
+pub struct BalancesData {
+    pub asset_id: AssetId,
+    pub account_id: AccountId,
+    pub free: u128,
```

Comment:
I think it would be better to use Balance type (https://github.com/Polkadex-Substrate/polkadexTEE-worker/blob/develop/polkadex-primitives/src/types.rs#L27)

Reply:
Done.